ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands.java

```diff
@@ -1397,7 +1396,7 @@ public void testNonAcidToAcidConversion01() throws Exception {
         BucketCodec.V1.encode(new AcidOutputFormat.Options(hiveConf).bucket(1)));
 
     //run Compaction
-    runStatementOnDriver("alter table "+ TestTxnCommands2.Table.NONACIDORCTBL +" compact 'major'");
+    runStatementOnDriver("alter table "+ Table.NONACIDORCTBL +" compact 'major'");
```

Comment:
nit: Maybe OCD but put spaces around the `+` signs please

Reply:
fixed